Formula/chakra.rb

```diff
@@ -12,23 +12,10 @@ class Chakra < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "icu4c" => :optional
 
   def install
-    # Build fails with -Os default
-    # Upstream issue from 26 Jan 2016 https://github.com/Microsoft/ChakraCore/issues/2417
-    # Fixed in master https://github.com/obastemur/ChakraCore/commit/cda81f4
-    ENV.O3
-
-    args = ["--static"]
-    if build.with? "icu4c"
-      args << "--icu=#{Formula["icu4c"].opt_include}"
-    else
-      args << "--no-icu"
-    end
-    system "./build.sh", *args
-
-    bin.install "BuildLinux/Release/ch" => "chakra"
+    system "./build.sh", "--lto-thin", "--static", "--embed-icu", "-j=#{ENV.make_jobs}", "-y"
```

Comment:
```
This script will download ICU-LIB from
http://source.icu-project.org/repos/icu/icu/tags/release-57-1
```
Is the checksum being verified? If not, can we vendor the resource instead in a resource block? Does it not work with our icu4c formula (58.2) and soon to be 59.1 (https://github.com/Homebrew/homebrew-core/pull/12461) yet?